pkg/da/data_availability_header_test.go

```diff
@@ -214,6 +214,47 @@ func Test_DAHValidateBasic(t *testing.T) {
 	}
 }
 
+func TestSquareSize(t *testing.T) {
+	type testCase struct {
+		name    string
+		dah     DataAvailabilityHeader
+		want    int
+		wantErr bool
+	}
+
+	testCases := []testCase{
+		{
+			name:    "min data availability header has an original square size of 1",
+			dah:     MinDataAvailabilityHeader(),
+			want:    1,
+			wantErr: false,
+		},
+		{
+			name:    "max data availability header has an original square size of 128",
+			dah:     maxDataAvailabilityHeader(),
+			want:    128,
```

Comment:
[Suggestion] I think it would be better to derive the hardcoded value of `128` from the `appconsts.DefaultSquareSizeUpperBound` instead (as a function of it), as the constants may change in the future.

Reply:
Resolved via https://github.com/celestiaorg/celestia-app/pull/2095/commits/8ef4bd6158cf8338f92a44f87e57a2cfcd80254b

Reply:
My suggestion was something like the following:
```
want: appconsts.DefaultSquareSizeUpperBound
```
instead of:

```
want: 128
```

Reply:
Sorry resolved via https://github.com/celestiaorg/celestia-app/pull/2095/commits/49dfa2e9271af22cb9ac5869ee378c9aa6d23670